packages/teams-js/src/public/media.ts

```diff
@@ -414,15 +416,16 @@ export namespace media {
      */
     protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): void {
       ensureInitialized(FrameContexts.content, FrameContexts.task);
-      const err = isApiSupportedOnMobile(nonFullScreenVideoModeAPISupportVersion);
-      if (err) {
+      try {
+        throwExceptionIfApiIsNotSupported(nonFullScreenVideoModeAPISupportVersion);
+      } catch (err) {
         if (callback) {
```

Comment:
If callback isn't defined this will fail silently. I would think we would only want the try catch here if callback was defined, and otherwise just call the throw function directly and let it throw up to the caller